node_modules/browser/web-viewer/actions.js

```diff
@@ -21,15 +22,29 @@ const initial = fromJS({
   icons: {},
   meta: {},
   backgroundColor: null,
-  securityState: 'insecure',
   securityExtendedValidation: false,
   canGoBack: false,
-  canGoForward: false
+  canGoForward: false,
+  theme: theme.defaultTheme
 });
 
-const open = () => initial.merge({id: uuid()});
+// Open new tab
+const open = () => initial.merge({
+  id: uuid()
+});
 exports.open = open;
 
+// Load initial tab.
+// @FIXME there may be a better way to handle this case: a more flexible `open`
+// function perhaps?
+const load = (uri) => initial.merge({
+  id: uuid(),
+  uri: uri,
+  isSelected: true,
+  isFocused: true
+});
+exports.load = load;
```

Comment:
I think I mentioned it in #77 as well, it would make sense to change open to:

```js
const open = (options={}) =>
  options.id ? initial.merge(options) : initial.merge(Object.assign(options, {id: uuid()}));
```

That way use of load(uri) will just be updated to `open({uri: 'http://...'})`